internal/elements.go

```diff
@@ -340,11 +340,7 @@ type GetETag struct {
 type ETag string
 
 func (etag *ETag) UnmarshalText(b []byte) error {
```

Comment:
This method is now a no-op and can be dropped.

Reply:
ok

Reply:
done